class.two-factor-core.php

```diff
@@ -320,7 +321,11 @@ public static function login_html( $user, $login_nonce, $redirect_to, $error_msg
 				<input type="hidden" name="provider"      id="provider"      value="<?php echo esc_attr( $provider_class ); ?>" />
 				<input type="hidden" name="wp-auth-id"    id="wp-auth-id"    value="<?php echo esc_attr( $user->ID ); ?>" />
 				<input type="hidden" name="wp-auth-nonce" id="wp-auth-nonce" value="<?php echo esc_attr( $login_nonce ); ?>" />
-				<input type="hidden" name="redirect_to"   id="redirect_to"   value="<?php echo esc_attr( $redirect_to ); ?>" />
+				<?php   if ( $interim_login ) { ?>
```

Comment:
http://php.net/control-structures.alternative-syntax would improve readability of code